### ISSUE TITLE
Adding collision ? 

### ISSUE THREAD

**Issue by nitneuq33** [sur950/Flutter_SnakeGame_FlutterWeb#1]
Hello, it's possible to add collision with snake body ? 
thank you

**Comment by sur950** [comment]
@nitneuq33 Yes we can add this. Kindly clone the project and change the logic.

I don't have time to work with this enhancement. If you can do this, I am happy to merge the PR.

Thank you.
